crates/ruff_linter/src/rules/flake8_bugbear/rules/loop_iterator_mutated.rs

```diff
@@ -0,0 +1,185 @@
+use ruff_diagnostics::Violation;
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::{
+    visitor::{self, Visitor},
+    Expr, ExprAttribute, ExprCall, ExprContext, ExprName, ExprSubscript, Stmt, StmtDelete, StmtFor,
+};
+use ruff_text_size::Ranged;
+
+use crate::checkers::ast::Checker;
+use ruff_diagnostics::Diagnostic;
+
+static MUTATING_FUNCTIONS: &'static [&'static str] = &[
+    "append", "sort", "reverse", "remove", "clear", "extend", "insert", "pop", "popitem",
+];
```

Comment:
We typically write this kind of thing as a method to enable the compiler to optimize it... Grep for `is_feature_name` as an example of what that looks like.

Reply:
Oh cool - I thought anyway that this seems to be not very rust-like.

Guess I've spent too much time in python :grimacing: